dotCMS/src/main/java/com/dotcms/rest/api/v2/languages/LanguagesResource.java

```diff
@@ -55,11 +56,19 @@ public LanguagesResource(final LanguageAPI languageAPI,
     /**
      * return a array with all the languages
      */
-    public Response  list(@Context final HttpServletRequest request) {
+    public Response  list(@Context final HttpServletRequest request, @QueryParam("contentInode") final String contentInode)
+            throws DotDataException, DotSecurityException {
+
         Logger.debug(this, String.format("listing languages %s", request.getRequestURI()));
```

Comment:
This can be replaced by  `Logger.debug(this, () -> String.format("listing languages %s", request.getRequestURI()));`
Using the supplier avoids the string creation when the logger is not in debug mode